RoddyCore/src/de/dkfz/roddy/execution/io/SSHExecutionService.groovy

```diff
@@ -110,18 +119,36 @@ class SSHExecutionService extends RemoteExecutionService {
 
             SSHClient c = new SSHClient()
             long t2 = System.nanoTime()
-            logger.postSometimesInfo(RoddyIOHelperMethods.printTimingInfo("create ssh client", t1, t2))
+
+            logger.sometimes(RoddyIOHelperMethods.printTimingInfo("create ssh client", t1, t2))
             try {
                 c.setConnectTimeout(1000)
                 c.addHostKeyVerifier(new PromiscuousVerifier())
                 c.connect(host)
                 t1 = System.nanoTime()
-                logger.postSometimesInfo(RoddyIOHelperMethods.printTimingInfo("connect ssh client", t2, t1))
+                logger.sometimes(RoddyIOHelperMethods.printTimingInfo("connect ssh client", t2, t1))
 
                 if (method == Constants.APP_PROPERTY_EXECUTION_SERVICE_AUTH_METHOD_PWD) {
+                    logger.always("Try setup the SSH connection ${user}@${host} using password authentification.")
```

Comment:
With "Try ..." it sounds like a imperative. Try "Trying ..." to indicate that this is ongoing work of the program.